src/roman_datamodels/datamodels/_datamodels.py

```diff
@@ -130,6 +130,58 @@ class ImageModel(_RomanDataModel):
 class ScienceRawModel(_RomanDataModel):
     _node_type = stnode.WfiScienceRaw
 
+    @classmethod
+    def from_tvac_raw(cls, model):
+        """Convert TVAC/FPS into ScienceRawModel
+
+        Parameters
+        ----------
+        model : ScienceRawModel, TvacModel, FpsModel
+            Model to convert from.
+
+        Returns
+        -------
+        science_raw_model : ScienceRawModel
+            The ScienceRawModel built from the input model.
+            If the input was a ScienceRawModel, that model is simply returned.
+        """
+        ALLOWED_MODELS = (FpsModel, ScienceRawModel, TvacModel)
+
+        if isinstance(model, cls):
+            return model
+        if not isinstance(model, ALLOWED_MODELS):
+            raise ValueError(f"Input must be one of {ALLOWED_MODELS}")
+
+        # Create base ramp node with dummy values (for validation)
+        from roman_datamodels.maker_utils import mk_level1_science_raw
+
+        raw = mk_level1_science_raw(shape=model.shape)
+
+        # Define how to recursively copy all attributes.
+        def node_update(raw, other):
```

Comment:
Does this address the schema differences?

For example, romancal expects `meta.guide_star.window_xstart`:
https://github.com/spacetelescope/romancal/blob/8422362dc33e0d4b3dc6702274b07c740acda37c/romancal/dq_init/dq_init_step.py#L55 (it's probably a bug that it gets this from the input and not the RampModel but either way).

This is ok for the current guidestar schema:
https://github.com/spacetelescope/rad/blob/50958b8991c38ab0071b8a1e9b2297e329444a1b/src/rad/resources/schemas/guidestar-1.0.0.yaml#L76
but the TVAC guidestar schema doesn't contain that data (it appears to be `gw_window_xstart`:
https://github.com/spacetelescope/rad/blob/50958b8991c38ab0071b8a1e9b2297e329444a1b/src/rad/resources/schemas/tvac/guidestar-1.0.0.yaml#L57

Reply:
This is a good point! Guiding stuff was not such a concern, but will go through and take a closer look. Got any good schema differs?

Reply:
I haven't used any schema differs before but would be curious if you find one you like.

Reply:
Well, no differs found. However, after discussion with Eddie, primary goal is to cover only absolutely necessary meta so that ELP will run. I believe those are covered. If not, they will be handled in other issues.